pages/_app.tsx

```diff
@@ -4,15 +4,21 @@ import { AppProps } from "next/app";
 
 import "styles/main.css";
 
+import ContextProviders from "contexts";
 import seoConfig from "config/seo";
+import Header from "components/Header";
 
 const App: React.FC<AppProps> = ({ Component, pageProps }) => (
   <>
     <Head>
       <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500;600&display=swap" rel="stylesheet" />
```

Comment:
I've removed the self-hosted fonts since they aren't working - I still don't know why 

But it's important to check if the fonts are taking too long to download and if it is - Let's try to solve the issue with the self-hosted fonts